deep_toggl/__init__.py

```diff
@@ -27,21 +28,35 @@ def summarize_all(config, date: pendulum.Date):
 
 def summarize_dispatch(project_name):
     today = pendulum.now()
-    config = get_config()
+    cfg = get_config()
 
     if project_name == "all":
```

Comment:
Magic string. Probably we need to declare it somewhere else

Reply:
`config` is the module, the linter raised a warning.
What do you suggest here?

Reply:
I don't get this :sweat_smile: This change was because the symbol `config` is related to a module and the linted warned me

Reply:
Ops I was referring to `all`

Reply:
I opened #17